spec/integration/lucky_cli_spec.cr

```diff
@@ -9,10 +9,21 @@ describe "Lucky CLI", tags: "integration" do
         shell: true,
         output: io
       )
-      status.exit_status.should eq(0)
+      status.exit_code.should eq(0)
```

Comment:
These changes are related to https://github.com/crystal-lang/crystal/issues/8381